deps/meck-0.7.2/src/meck.erl

```diff
@@ -71,7 +69,7 @@
 
 %% Records
 -record(state, {mod :: atom(),
-                expects :: dict(),
+                expects :: dict:dict(),
```

Comment:
Please try bumping deps to an ulterior version that compiles with Erlang ≥17 instead of patching them.